src/main/editDelete/__tests__/preProcess.test.ts

```diff
@@ -38,7 +38,7 @@ describe('Test pre-processing JSON transcript into regular transcript', () => {
           fileName: 'PLACEHOLDER FILENAME',
         },
         {
-          word: 'abc',
+          word: 'Abc.',
```

Comment:
Can you add a test that specifically tests for different punctuation types, I can't see any commas in the tests and I think we should have a punctuation specific test to make sure it's being tested explicitly

Reply:
Yep, no problem